### PR TITLE
fix(qannotate): snpeff records were not being dealt with correctly

### PR DESCRIPTION
**Pull request by holmeso**
# Description

If a vcf record being annotated has more than 1 entry in the alt field, then nanno will split that record into 2 records.
eg.
`chr1 1000 A C,T .....`
will become
`chr1 1000 A C .....`
and
`chr1 1000 A T .....`

However, nanno was using the whole of the alt field in snpEff annotation files, which meant that there would not be a match from snpEff (just for records that had more than 1 entry in the alt field).

The fix involves splitting the 'alt' field when it contains more than one value, allowing the VCF record to be annotated to match against each of these values.

Other changes:
 - reverted a change made be the previous PR #341 that removed a build.last step
 - renamed ChePositionRefAlt.getName() to getRef()
 - added GATK GT field to nano output
 - added original_alt field to nanno output (useful when splitting 1/2 variants)
 - keep original GATK_AD values rather than manipulating them (original_alt field allows for this)
 - update Executor so that spaces in the classpath (hello IDEA) don't cause the Process to fall over
 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

New unit tests, along with testing on cluster

# Are WDL Updates Required?

No

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: q3vcftools/src/au/edu/qimr/vcftools/AmalgamatorGS.java [hunk 1]
```diff
@@ -169,7 +169,7 @@ private void writeOutput() throws FileNotFoundException {
 				}
 				String scoreS = score + "/" + (4 + vcfFiles.length);
 				scoreDist.computeIfAbsent(scoreS, v -> new AtomicInteger()).incrementAndGet();
-				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + Constants.TAB +cp.getName() + Constants.TAB 
+				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + Constants.TAB +cp.getRef() + Constants.TAB
 						+ cp.getAlt() + Constants.TAB + Arrays.stream(p.getLeft()).map(s -> null == s ? "./." : s).collect(Collectors.joining(Constants.TAB_STRING))
 						+ Constants.TAB +  Arrays.stream(p.getRight()).map(s -> null == s ? "." : s).collect(Collectors.joining(Constants.TAB_STRING))
 						+ Constants.TAB +  Arrays.stream(missingACs).map(s -> null == s ? "." : s).collect(Collectors.joining(Constants.TAB_STRING))
```
Comment (delocalizer) [inline q3vcftools/src/au/edu/qimr/vcftools/AmalgamatorGS.java:175]: Minor query on consistency — is there a rationale for using `TAB` and `TAB_STRING` from `Constants` but not `SLASH_STRING`, `MISSING_DATA`, `MISSING_GT` etc. for the other string literals here?

File: q3vcftools/src/au/edu/qimr/vcftools/Classify.java [hunk 2]
```diff
@@ -162,7 +162,7 @@ private void writeOutput(List<ChrPositionRefAlt> recs, String output) throws Fil
 			ps.println(header);
 			for (ChrPositionRefAlt cp : recs) {
 //				logger.info("writing out: " + cp.getName());
-				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + "\t.\t" + cp.getName() + Constants.TAB + cp.getAlt() + "\t.\t.\t.");
+				ps.println(cp.getChromosome() + Constants.TAB + cp.getStartPosition() + "\t.\t" + cp.getRef() + Constants.TAB + cp.getAlt() + "\t.\t.\t.");
```
Comment (delocalizer) [inline q3vcftools/src/au/edu/qimr/vcftools/Classify.java:165]: While we're here, is this an opportunity to generate output in a nicer (more explicit) way, given that these look basically like VCF records? Essentially the same concatenation is used a couple more times below too.

**Review by delocalizer — APPROVED** [review]

File: q3vcftools/src/au/edu/qimr/vcftools/Classify.java (hunk 2 above)
Comment (holmeso) [reply q3vcftools/src/au/edu/qimr/vcftools/Classify.java]: added a method to ChrPositionUtils that takes a ChrPosition (and some other bits of info) and returns a Vcf string

File: q3vcftools/src/au/edu/qimr/vcftools/AmalgamatorGS.java (hunk 1 above)
Comment (holmeso) [reply q3vcftools/src/au/edu/qimr/vcftools/AmalgamatorGS.java]: Complete lack of consistency - using more Constatns now (even using static imports)

**Pull request merged**